src/helpers/clipboard/clipboard_os_state.ts

```diff
@@ -18,18 +19,49 @@ export class ClipboardOsState extends ClipboardCellsAbstractState {
   constructor(
     content: string,
```

Comment:
`paste` of `ClipboardOsState` shouldn't be the same as paste of the cell handler ? They both are pasting cells after all.
I believe the `content` (raw content of the os clipboard) should be converted to the same data structure as what the cell handler `copy` would give.